src/main.rs

```diff
@@ -66,7 +63,6 @@ fn main() -> Result<(), aes_gcm_siv::Error> {
         // b_i <-- KA.R
```

Comment:
can do `for item in SET_Y` since we don't need the index. or if you want to be reeeal rusty use the iter utils and do a `set.map(<prime gen logic here>).collect()` cuz at the end of the day what ur doing here is taking a length n iterable and producing a length n iterable. this is what map is for